### DISCUSSION TITLE
Reordering columns & rows (drag + drop & arrows in the menu)

### DISCUSSION THREAD

**Discussion by almereyda** [outline/outline#7626] [Ideas]
Currently, it is not possible to reorder columns or rows in a table, either by dragging and dropping the large selection area, nor with dedicated buttons.

We know how to add columns and how to sort them, but not how to rearrange them.

![grafik](https://github.com/user-attachments/assets/eb451e89-47ef-4dbc-bc5f-cdd2a8b383f4)

This would make it very convenient to work with tables in an incremental way.

**Comment by adrfantini** [comment]
I think this is now possible and can be closed:
<img width="259" height="200" alt="image" src="https://github.com/user-attachments/assets/828c7335-85fd-45d2-8275-a0a010a34890" />
